### ISSUE TITLE
Separate cropping from other operations in the augmentor

### ISSUE THREAD

**Issue by NeelayS** [neu-vig/ezflow#72]
For the validation set, we only want to do cropping and not any other augmentations. It would thus be a good idea to separate cropping from the rest of the operations. By doing this, we will be able to pass the "augment" argument as False in the dataloader creator for the validation set. Currently, to ensure cropping but not other augmentation operations for the validation set, we have to manually set the probabilities of all the augmentation operations to 0.
